TOC.md

```diff
@@ -107,7 +107,7 @@
         + [SQL Plan Management](/sql-plan-management.md)
         + [Access Tables Using `IndexMerge`](/index-merge.md)
 + Tutorials
-  + [Geo-Redundant Deployment](/geo-redundancy-deployment.md)
+  + [Multi-DC in One City Deployment](/multi-data-centers-in-one-city-deployment.md)
```

Comment:
I see that in https://github.com/pingcap/docs/pull/2963 you used the full name of data center. Let's keep them consistent.
```suggestion
  + [Multiple Data Centers in One City Deployment](/multi-data-centers-in-one-city-deployment.md)
```